pkg/apis/serving/v1beta1/route_types.go

```diff
@@ -89,10 +89,17 @@ type TrafficTarget struct {
 	// +optional
 	LatestRevision *bool `json:"latestRevision,omitempty"`
 
-	// Percent specifies percent of the traffic to this Revision or Configuration.
-	// This defaults to zero if unspecified.
+	// Percent indicates that percentage based routing should be used and
+	// the value indicates the percent of traffic that is be routed to this
+	// Revision or Configuration. `0` (zero) mean no traffic, `100` means all
+	// traffic.
+	// If "percent" appears on more than one Traffic Target then the sum of
+	// all those percent values must equal 100.
+	// If "percent", nor any other routing mechanism is specified, then
+	// the default is that no traffic will be sent to this Revision or
+	// Configuration.
 	// +optional
-	Percent int `json:"percent"`
+	Percent *int64 `json:"percent"`
```

Comment:
Considering it's percent (0-100) why int64? And not, say, int32?

Reply:
No particular reason. I can do a global replace if people really want me to.